config/autoload/commercial-vendors.global.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    'commercial-vendors' => [
+        [
+            'name' => 'Roave',
+            'logo' => '/images/vendors/roave.png',
+            'url' => 'https://roave.com',
+            'text' => 'Roave is a full-service web development firm, offering services such as consulting, training, software development, and more.',
```

Comment:
I suggest removing the second "services" as "service" was already used earlier in the sentence. How about: "Roave is a full-service web development firm, offering consulting, training, software development, and more."